demo/MINDCUB3R/README.md

```diff
@@ -1,26 +1,34 @@
 # MINDCUB3R
 
 ### Installation
-This can take a minute on an EV3
-* cd ~/ev3dev-lang-python/demo/MINDCUB3R/kociemba/ckociemba/
-* sudo apt-get update
-* sudo apt-get install build-essential
-* make
-* sudo make install
-
-The first time you run this it creates a cache directory, this takes about 30s on an EV3
-
-* cd ~/ev3dev-lang-python/demo/MINDCUB3R/
-* kociemba DRLUUBFBRBLURRLRUBLRDDFDLFUFUFFDBRDUBRUFLLFDDBFLUBLRBD
-
+This can take several minutes on an EV3
+```
+sudo apt-get install build-essential python3-pip libffi-dev
+sudo pip3 install git+https://github.com/dwalton76/kociemba.git
+sudo pip3 install git+https://github.com/dwalton76/rubiks-color-resolver.git
+kociemba DRLUUBFBRBLURRLRUBLRDDFDLFUFUFFDBRDUBRUFLLFDDBFLUBLRBD
+```
 
 ### Running
 ./rubiks.py
 
-
-### Credit
-Much of rubiks.py and rubiks_rgb_solver.py were ported from
-https://github.com/cavenel/ev3dev_examples
-
-The kociemba code is from
-https://github.com/muodov/kociemba
+### About kociemba
+You may have noticed that the
+`kociemba DRLUUBFBRBLURRLRUBLRDDFDLFUFUFFDBRDUBRUFLLFDDBFLUBLRBD`
+step of the install looks a little odd. The "DRLUU..." string is a
+representation of the colors of each of the 54 squares of a 3x3x3 cube. So
+the D at the beginning means that square #1 is the same color as the middle
```

Comment:
This will probably auto-link to the GitHub issue #1; you can put it in code formatting if you want or just leave it as-is.